packages/formstr-app/src/containers/FormFillerNew/QuestionNode/InputFiller.tsx

```diff
@@ -43,13 +45,15 @@ export const InputFiller: React.FC<InputFillerProps> = ({
           defaultValue={defaultValue as string}
           onChange={handleInputChange}
           placeholder="Please enter your response"
+          data-testid={testId}
```

Comment:
we can do something like `${testId}:input-filler`

so the parent can send some test id and it would be namespaced by the child component automatically

Reply:
rest looks fine